converter/src/main/java/gov/cms/qpp/conversion/encode/QualityMeasureIdEncoder.java

```diff
@@ -3,6 +3,7 @@
 import gov.cms.qpp.conversion.Context;
 import gov.cms.qpp.conversion.decode.AggregateCountDecoder;
 import gov.cms.qpp.conversion.decode.MeasureDataDecoder;
+import gov.cms.qpp.conversion.decode.ReportingParametersActDecoder;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'gov.cms.qpp.conversion.decode.ReportingParametersActDecoder'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)